templates/partners/become-a-partner.html

```diff
@@ -103,7 +103,7 @@ <h3 class="p-muted-heading">Partnering with Canonical gives you:</h3>
       </ul> #}
     </div>
     <div class="col-6 col-start-large-7">
-      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_1260">
+      <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_1260" onsubmit="ga('send', 'Become a partner', 'Signup', 'Become a partner application');">
```

Comment:
It works, but should we be submitting it to a different domain still?

Reply:
mm this is what we are doing at https://github.com/canonical/canonical.com/blob/9889a905623e26b2cb487f094d9b7d6e28f8c858/templates/blog/newsletter-form.html#L7

Do we need to create the endpoint in this project? It would create duplication in many places.

Reply:
Could we turn it into a module to use across projects? I think this site is having the same issue: https://github.com/canonical/web-squad/issues/6146

Reply:
Well, I can discuss it with the infra team, but I think there could be other ideas, e.g. Joao suggested a service for discourse so this could be part of that. For now, I think the ubuntu.com one works well?

Reply:
Created issue to discuss this https://github.com/canonical/web-squad/issues/6241